tests/test_add_nightreport.py

```diff
@@ -0,0 +1,75 @@
+# This file is part of ts_nightreport.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import httpx
+import psycopg
+import pytest
+from lsst.ts.nightreport.testutils import (
+    ReportDictT,
+    assert_good_response,
+    cast_special,
+    create_test_client,
+)
+
+
+def assert_good_add_response(response: httpx.Response, add_args: dict) -> ReportDictT:
+    """Check the response from a successful add_reports request.
+
+    Parameters
+    ----------
+    response
```

Comment:
Please specify the type of the parameters `response` and `add_args` and of the return type.